Invoke-DiagnosticCheck.ps1

```diff
@@ -157,6 +157,10 @@ Describe "Diagnostic check" {
             }
             $true | Should Be $true
         }
+
+        It "npcap should not be installed" {
+            Get-Service npcap | Should Throw
```

Comment:
For testing if something throws in Pester, you must wrap the LHS in a script block, like so:

```
{Get-Service npcap} | Should Throw
```

See the `Warning` here: https://github.com/pester/Pester/wiki/Should#throw

Reply:
Also, we should add a comment here explaining why it should not be installed